superset-frontend/src/views/CRUD/alert/AlertList.tsx

```diff
@@ -275,6 +285,49 @@ function AlertList({
     slot: canCreate ? EmptyStateButton : null,
   };
 
+  const filters: Filters = useMemo(
+    () => [
+      {
+        Header: t('Created By'),
+        id: 'created_by',
+        input: 'select',
+        operator: 'rel_o_m',
```

Comment:
I add these to an enum a recently (https://github.com/apache/incubator-superset/blob/1842c56209131757f47bf7706a601aa933c6da4b/superset-frontend/src/components/ListView/types.ts#L103) can we use that as we do in `QueryList` which should make their meaning a little less ambiguous.